flame/values.yaml

```diff
@@ -1,8 +1,15 @@
 # Global values
 global:
-  hubAuth:
-    robotUser: ""
-    robotSecret: ""
+  hub:
+    endpoints:
+      auth: https://auth.privateaim.dev
+      core: https://core.privateaim.dev
+      messenger: https://messenger.privateaim.dev
+      storage: https://storage.privateaim.dev
+    auth:
+      realmId: ""
+      robotUser: "1"
```

Comment:
Shouldn't we leave this as an empty string? The hub adapter and message broker both set this as "required" so we don't want it to think this is a valid default value

Reply:
Thanks, yes, my bad, I'll fix it.